css/styles.css

```diff
@@ -0,0 +1,8 @@
+* {
+  box-sizing: border-box;
+  margin: 0;

```

Comment:
You should add more reset styles: for buttons, ul, a... This is not enough, in my opinion. What do you think?

Reply:
You are right, here you can see the changes: https://github.com/AnitaMartinez/clase-git-colaborativo/pull/1/commits/6fdbef2efc1f3cfc1b70abc8f9133488792520a9

Reply:
That's perfect!